apps/service-portal/src/hooks/useRoutes/useRoutes.ts

```diff
@@ -3,12 +3,14 @@ import { useEffect } from 'react'
 import { ServicePortalModule } from '@island.is/service-portal/core'
 import { Action, ActionType } from '../../store/actions'
 import { useModuleProps } from '../useModuleProps/useModuleProps'
+import useModules from '../useModules/useModules'
```

Comment:
Can you change this to a named export, so the name won't differ between files.

Reply:
I was following the same naming pattern for most of the other hooks. Is this some standard that we use?

Reply:
But the hook above is a named export. This mainly comes from https://basarat.gitbook.io/typescript/main-1/defaultisbad and is the reason why we almost always reexport our default imports in an index file.
```typescript
// index.ts
export { default as useModules } from './useModules'
```

```typescript
// apps/service-portal/src/hooks/useRoutes/useRoutes.ts
import { useModules } from '../useModules'
```

Reply:
Yeah they were mixed this was the only hook with named export and all the others had default export so I opted to be consistent.

But is this is a standard I will change this to named export. Someone else needs to refactor the other hooks and put in the interface